ddtrace/internal/processor/trace.py

```diff
@@ -126,6 +126,7 @@ def process_trace(self, trace):
             return trace
 
         ctx._update_tags(chunk_root)
+        chunk_root.set_tag_str("language", "python")
```

Comment:
Is `language` a trace level tag? If so we should reorder these lines:

```suggestion
        chunk_root.set_tag_str("language", "python")
        ctx._update_tags(chunk_root)
```

Reply:
not sure we want this to be over-rideable?

Reply:
On second thought is probably a bad idea. A trace can contain chunks/spans from different languages. Depending on how the agent sets trace level tags this could overwrite values set by other libraries. We should probably add comment stating this.

Reply:
I am not sure I follow your line of thinking.

Reply:
@mabdinur how can a trace contain spans from multiple languages?

Reply:
I think the order looks correct, and I'm not aware of any special agent/backend trace level tags magic that occurs regarding whatever tags are set on the local root span, which in this case I'm pretty sure will only ever be a Python span.

Reply:
> trace level tags magic

👍🏻 correct, they'd only be for the portion of the trace from this process. we don't have any way to set a trace "global" (?) tag.

Reply:
Yup, I was mistaken. I thought we submitted "tracer level tags" to the agent but this is not the case. I'll resolve this thread and my suggestion